README.md

```diff
@@ -1,50 +1,76 @@
+1. **Facilitate Open Communication**: Provide a place for contributors and community members to propose, discuss, and refine new ideas or changes to Meshtastic in an open and inclusive manner.
+1. **Ensure Transparency**: Make the decision-making process visible to all stakeholders, allowing for a transparent and democratic approach to Meshtastic development.
+1. **Promote Collaboration**: Encourage contributions to leverage the collective expertise and perspectives of our community to drive Meshtastic forward.
+1. **Maintain Project Integrity**: Ensure that changes and new features align with Meshtastic's overall goals and values, while preserving integrity and focus.
 
-## How to Make an RFC
+## Contribution Guidelines & Participation
 
-1. **Familiarize with the RFC Template**: Before submitting an RFC, please review our [RFC Template](template.md). This template provides a structured format for proposing changes.
+While contributing to the Meshtastic RFC repository, please adhere to our community standards and guidelines. These are in place to ensure a productive and respectful environment for collaboration.
 
-1. **Write up your proposal**: There are two methods to write up your proposal within this repository. If you are comfortable with how to work with Git, we recommend you clone this repository, create a copy of the RFC template in the `/rfcs` directory, and submit a PR. If you are not familiar with Git, we recommend you follow the steps below:
+- Be respectful and constructive in your communication.
+- Clearly articulate your proposals and feedback.
+- Engage in discussions with an open mind.
+- **Informal Discussions**: Before submitting an RFC, you may want to discuss your idea informally. Join us on our [Discord](https://discord.com/invite/ktMAKGBnBs) for preliminary discussions and feedback from the community.
 
-   1. Sign into or create a GitHub account. Then, navigate to [https://github.com/meshtastic/rfcs](https://github.com/meshtastic/rfcs).
+## Applicability
 
-   1. Navigate to the RFC template in the root of the repository. Copy the contents of the template to your clipboard.
+### When an RFC is Required
 
-   1. Navigate to the `/rfcs` directory within this repository on GitHub. Click the `Add file` button and select `Create new file`. Name this file `YYYY-MM-DD-feature-name.md`, replacing `YYYY-MM-DD` with the current date and `feature-name` with a descriptive name for your proposal. Paste the contents of the template into this file.
+An RFC is required for any significant change to Meshtastic, introduces new features, or has the potential to impact the user experience or ecosystem broadly. This includes, but is not limited to:
```

Comment:
The scope of this feels vague enough to where someone might assume that any new feature period would require an RFC. Maybe we need to call out that it's really anything with significant implementation impact across the platforms?

Reply:
Yes I agree here, the `introduces new features` is too broad. How do we define what `significant` means? Anyone have ideas?

Reply:
> Yes I agree here, the `introduces new features` is too broad. How do we define what `significant` means? Anyone have ideas?

Non-backward compatible?

Reply:
```
An RFC is required for any change that breaks backwards compatibility or would broadly impact either the User Experience or Ecosystem of Meshtastic. This includes but is not limited to:

- Changes to the core architecture
- Major UI/UX changes (e.g. Renaming Channels or Nodes)
- Changes affecting backward compatibility
```

Reply:
> ```
> An RFC is required for any change that breaks backwards compatibility or would broadly impact either the User Experience or Ecosystem of Meshtastic. This includes but is not limited to:
> 
> - Changes to the core architecture
> - Major UI/UX changes (e.g. Renaming Channels or Nodes)
> - Changes affecting backward compatibility
> ```

I like it. Maybe use a word like recommended or warranted possibly instead of required? Makes it seem like there could be punitive consequences if something is done that is perceived as a significant change that doesn't go through an RFC.

Reply:
This brings up a good question, what would happen if a change goes through that should have been an RFC? Does it get backed out?